src/platform/site-wide/sass/minimal.scss

```diff
@@ -2,6 +2,7 @@
 @import "modules/m-overrides";
 
 @import "~uswds/src/stylesheets/core/variables";
+@import "~uswds/src/stylesheets/core/base";
```

Comment:
This applies the [focus mixin](https://github.com/uswds/uswds/blob/c4c9410d54b537c6112037a36ed289d8b7e8940a/src/stylesheets/core/_base.scss#L39-L41).